src/sentry/integrations/slack/message_builder/base/block.py

```diff
@@ -41,19 +41,13 @@ def get_markdown_block(text: str, emoji: str | None = None) -> SlackBlock:
         }
 
     @staticmethod
-    def get_rich_text_preformatted_block(text: str) -> SlackBlock:
+    def get_markdown_quote_block(text: str) -> SlackBlock:
         if len(text) > MAX_BLOCK_TEXT_LENGTH:
             text = text[: MAX_BLOCK_TEXT_LENGTH - 3] + "..."
-        return {
-            "type": "rich_text",
-            "elements": [
-                {
-                    "type": "rich_text_preformatted",
-                    "elements": [{"type": "text", "text": text}],
-                    "border": 0,
-                }
-            ],
-        }
+
+        markdown_text = "```" + text + "```"
```

Comment:
Should we have a newline after the first ``` block, and before the last? Was just a little worried slack might think "```abc" is the actual text haha

Reply:
well, I guess that answered my question, we're good haha

Reply:
lol are you talking about escaping the backticks

Reply:
I was worried slack would think ```abc is a full word and not start the block quote

Reply:
so if we have a newline "```\n", then maybe it would be safer in starting the block quote